GDJS/Runtime/scenestack.ts

```diff
@@ -385,5 +472,47 @@ namespace gdjs {
         throw 'The scene stack has been disposed and should not be used anymore.';
       }
     }
+
+    loadGameFromIndexedDB = async function (
```

Comment:
This probably should not exist in the scenestack.ts as it does not have anything to do with scenes?

You can create a file named `indexeddb.ts` in the runtime folder with those generic indexedDB methods `loadFromIndexedDB(dbName, key, storename)`
as well as `saveToIndexedDb(dbName, key, storename, data)` (the one used in your extension)

And use them in your files. You can take a look at `capturemanager.ts` for inspiration